wardenjs/proto/ethermint/evm/v1/evm.proto

```
@@ -3,12 +3,14 @@
 syntax = "proto3";
 package ethermint.evm.v1;
 
+import "amino/amino.proto";
```

buf

import "amino/amino.proto": file does not exist

(COMPILE)